### DISCUSSION TITLE
能不能使用golang 实现一个cluster，让skynet 和它通讯？

### DISCUSSION THREAD

**Discussion by brucelin-cn** [cloudwu/skynet#1952] [Ideas]
如题，目前我已有好几个golang的服务，现有skynet服务都是通过http 互相访问的，如果http 改为 cluster访问的话，应该能提高不少性能吧？至少是方向是skeynet->golang

我的想法初步想法，需要clustersender 支持另外的skynet.dispatch(typename, func)，现有的是skynet.dispatch(“lua”,..)。

**Comment by hanxi** [comment]
已经有人实现过 https://github.com/xingshuo/skyline

**Comment by terry8210** [comment]
这个不是更直接吗？https://github.com/Zwlin98/moon

**Reply by brucelin-cn** [reply]
确实，这个实现的更直接。
